shard/src/test/java/com/redhat/service/bridge/shard/ManagerSyncServiceTest.java

```diff
@@ -90,9 +90,10 @@ public void testProcessorsAreDeployed() throws Exception {
         ProcessorDTO processor = createProcessor(bridge, BridgeStatus.REQUESTED);
 
         stubProcessorsToDeployOrDelete(Collections.singletonList(processor));
+        stubProcessorUpdate();
```

Comment:
Not really related to the tests, but it removes some warning messages because an endpoint is not stubbed